GooglePlayInstant/Editor/PlayInstantQuickDeployWindow.cs

```diff
@@ -14,15 +14,20 @@
 
 using UnityEditor;
 using UnityEngine;
+using System.Diagnostics;
```

Comment:
Is this still needed? Run the IDE's Code -> Optimize Imports option on each file you modified.

Reply:
Resolved.